### PR TITLE
Add GitHub Oauth guide

### PR DESCRIPTION
**Pull request by rtomas**
here is the example PR
https://github.com/Web3Auth/web3auth-pnp-examples/pull/922

### CODE REVIEW

File: src/pages/guides/github-oauth.mdx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  title="Integrating Github OAuth with Web3Auth for Authentication"
+  description="Learn how to integrate Github OAuth with Web3auth."
+  image="https://web3auth.io/docs/guides/banners/github.png"
+  slug="/guides/github-oauth"
+/>
+
+This guide will cover the basics of how to integrate Web3Auth with GitHub OAuth for authentication. Web3Auth is employed to provide Ethereum private
+key access and public address and GitHub OAuth is used to authenticate users
+
+Full example: https://github.com/web3auth/web3auth-byoa/github-oauth-connection
```
Comment (shahbaz17) [inline src/pages/guides/github-oauth.mdx:25]: This link is broken

File: src/pages/guides/microsoft-oauth.mdx [hunk 2]
```diff
@@ -15,8 +15,8 @@ import SEO from "@site/src/components/SEO";
 <SEO
   title="Integrating Microsoft OAuth with Web3Auth for Decentralized Authentication"
   description="Learn how to integrate Microsoft Github OAuth with Web3auth."
-  image="https://web3auth.io/docs/content-hub/guides/banners/microsoft.png"
-  slug="/content-hub/guides/microsoft-github-oauth"
+  image="https://web3auth.io/docs/guides/banners/microsoft.png"
```
Comment (shahbaz17) [inline src/pages/guides/microsoft-oauth.mdx:18]: please change the image and slug to match github image

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

File: src/pages/guides/github-oauth.mdx (hunk 1 above)
Comment (rtomas) [reply src/pages/guides/github-oauth.mdx]: solve

**Review by shahbaz17 — APPROVED** [review]
lgtm

**Pull request merged**